pretty_printers_tests/lldb_batchmode.py

```diff
@@ -87,6 +87,9 @@ def execute_command(command_interpreter, command):
     if res.Succeeded():
         if res.HasResult():
             output = res.GetOutput() or ''
+            if sys.platform == 'win32':
```

Comment:
Since it's quite unexpected (at least for me, who is not a python developer) that `win32` is returned for both x32 and x64 platform, let's add the corresponding comment with https://docs.python.org/3/library/sys.html#sys.platform link